objcgen/objcgenerator-postprocessor.cs

```diff
@@ -24,6 +25,11 @@ protected IEnumerable<ProcessedMethod> PostProcessMethods (IEnumerable<MethodInf
 
 				if (method.IsSpecialName && method.IsStatic && method.Name.StartsWith ("op_", StringComparison.Ordinal))
 					processedMethod.IsOperator = true;
+				
+				if (operatorToIgnore.Contains (method)) {
+					processedMethod.Ignore = true;
```

Comment:
why not `continue;` ? (and move the delayed warning just before)

is there a case where we should give an _ignored_ `ProcessedMethod` further down ?